src/app/pages/settings/categories/layout.component.ts

```diff
@@ -9,21 +9,30 @@ tmplLayout.innerHTML = `
 		</div>
 		<div class="accordion-collapse collapse" data-bs-parent="#categories">
 			<div class="accordion-body px-3">
+				<div class="c-category__settings-container d-flex flex-column gap-2 mb-3">
+					<app-margin-align class="c-layout__setting" data-can-edit="true"></app-margin-align>
+				</div>
+				<button class="c-category__btn-more btn btn-tertiary" type="button" data-i18n="moreSettings"></button>
 			</div>
 		</div>
 	</div>
 `;
 
 class LayoutComponent extends AbstractCategory {
-
 	constructor() {
-		let settingsDictionnary: any[] = [
+		const settingsDictionnary: any[] = [
+			{ name: 'marginAlign', element: 'app-margin-align' },
 		];
 
 		super(settingsDictionnary);
 
 		this.appendChild(tmplLayout.content.cloneNode(true));
 	}
+
+	connectedCallback(): void {
+		let settingsElements = [...this.querySelectorAll('.c-layout__setting')];
```

Comment:
Je viens de tiquer là-dessus maiqs j’en avais vu ailleurs je crois : pourquoi cette syntaxe ? Ici tu ngas qu’un seul `.c-layout__setting`, tu pourrais te contenter d’un `this.querySelector`, non ?

Reply:
C'est pour anticiper les prochains qui arriveront. Vu que la classe sera appliqué sur chaque réglage d'une catégorie, plutôt que de faire this.querySelector ici et changer quand je passerai à 2 réglages, j'ai préféré faire directement ça. Si ça pose soucis je peux revenir à ce que tu proposes.